code/API_definitions/kyc-match.yaml

```
@@ -199,79 +199,79 @@ components:
-
+          description: First/given name or compound first/given name of the customer.
+          
         familyName:
           type: string
-          description: Surname/family name of the customer. It may include a compound last name or an additional last name.
-
+          description: Last name, family name, or suranme of the customer.
+          
         nameKanaHankaku:
           type: string
-          description: Reading of the full name of the customer in full-width Kana format.
+          description: Complete name of the customer in Hankaku-Kana format (reading of name) for Japan.
 
         nameKanaZenkaku:
           type: string
-          description: Reading of the full name of the customer in half-width Kana format.
+          description: Complete name of the customer in Zenkaku-Kana format (reading of name) for Japan.
 
         middleNames:
           type: string
-          description: Middle names of the customer
+          description: Middle name/s of the customer.
 
         familyNameAtBirth:
           type: string
-          description: Family name at birth of the customer
+          description: Last/family/sur- name at birth of the customer.
 
         address:
           type: string
-          description: Address of the customer
-
+          description: Complete address of the customer.  For some countries, it is built following the usual concatenation of parameters in a country, but for other countires, this is not the case.  For some countries, it can use streetName, streetNumber and/or houseNumberExtension. For example, in ESP, streetName+streetNumber; in NLD, it can be streetName+streetNumber or streetName+streetNumber+houseNumberExtension.
+          
         streetName:
           type: string
-          description: Name of the street or other street type. It should not include street type
+          description: Name of the street of the customer's address. It should not include the type of the street.
 
         streetNumber:
           type: string
-          description: Generally a number identifying a specific property on the `street_name`, but it may be a range of numbers (10-12) or include some letter (10b)
+          description: The street number of the customer's address.  Number identifying a specific property on the 'streetName'.
 
         postalCode:
           type: string
           description: Zip code or postal code
 
         region:
           type: string
-          description: Regin/prefecture of the customer's address
+          description: Region/prefecture of the customer's address
 
         locality:
           type: string
           description: Locality of the customer's address
 
         country:
           type: string
-          description: Country of the customer's address
+          description: Country id of the customer's address, in ISO 3166 alpha-3 format.

```

Hi @ToshiWakayama-KDDI !
In other CAMARA APIs (such as Device Status) alpha-2 is being used, we should stick to it for consistency. Alpha-2 is also a more widespread format . Maybe this could be an interesting topic to discuss in commonalities.

```suggestion
          description: Country id of the customer's address, in ISO 3166 alpha-2 format.

```